modules/apps/portal-search/portal-search-admin-web/src/main/java/com/liferay/portal/search/admin/web/internal/display/context/builder/SearchAdminDisplayContextBuilder.java

```diff
@@ -53,23 +52,21 @@ public SearchAdminDisplayContext build() {
 			themeDisplay.getPermissionChecker();
 
 		if (permissionChecker.isOmniadmin()) {
-			selectedTab = getSelectedTab();
-
 			_addNavigationItemList(
-				navigationItemList, "connections", selectedTab);
+				navigationItemList, "connections", getSelectedTab());
 		}
 
 		_addNavigationItemList(
-			navigationItemList, "index-actions", selectedTab);
+			navigationItemList, "index-actions", "index-actions");
 
 		if (_isIndexInformationAvailable() && permissionChecker.isOmniadmin()) {
 			_addNavigationItemList(
-				navigationItemList, "field-mappings", selectedTab);
+				navigationItemList, "field-mappings", "index-actions");
 		}
 
 		searchAdminDisplayContext.setNavigationItemList(navigationItemList);
 
-		searchAdminDisplayContext.setSelectedTab(selectedTab);
+		searchAdminDisplayContext.setSelectedTab("index-actions");
```

Comment:
im not understanding this logic change (the commit message just says SF?). why is "index-actions" always the selected tab now?

Reply:
I thing I made a wrong refractor 😅